Replace ClusterRole from Helm chart with dependent Role to allow Pod annotation updates when the operator is not running (#240)
hivemq/helm-charts: charts/hivemq-platform-operator/templates/NOTES.txt

{{- if ne .Values.serviceAccount.create true -}}

*** Warning **
The ServiceAccount {{ .Values.serviceAccount.name }} should have correct permissions for the Operator to work correctly.
{{- end }}
{{- if ne .Values.platformServiceAccount.create true -}}

*** Warning **

> The Operator will not create a ServiceAccount for the HiveMQ Platform Pods.

Reviewer: ```suggestion
The Operator is not configured to create a ServiceAccount for the HiveMQ Platform Pods.
```

Please make sure that you configure a valid custom ServiceAccount in the HiveMQ Platform values.yaml (see nodes.serviceAccountName).
{{- end }}
{{- if ne .Values.platformServiceAccount.permissions.create true -}}

*** Warning **

> The Operator will not create the Role and RoleBinding for the ServiceAccount of the HiveMQ Platform Pods.

Reviewer: ```suggestion
The Operator is not configured to create the Role and RoleBinding for the ServiceAccount of the HiveMQ Platform Pods.
```

> Please make sure that the ServiceAccount that is configured in the HiveMQ Platform values.yaml (see nodes.serviceAccountName) has the correct RBAC permissions to list, get and watch Pods.

Reviewer: ```suggestion
Please make sure the HiveMQ Platform values.yaml configures a ServiceAccount (see nodes.serviceAccountName) and has the correct RBAC permissions to list, get and watch Pods.
```
